### PR TITLE
[HOTFIX] Allow space between input file paths

### PR DESCRIPTION
**Pull request by kevinjmh**
### Why is this PR needed?
 When space exists between file paths, `The input file does not exist` is thrown.

```
LOAD DATA INPATH 'hdfs:///data/file1.dat , hdfs:///data/file2.dat' INTO TABLE ...
```
 
 ### What changes were proposed in this PR?
trim space of each file path before usage
    
 ### Does this PR introduce any user interface change?
 - No
 - Yes. (please explain the change and update document)

 ### Is any new testcase added?
 - No
 - Yes

### CODE REVIEW

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/155/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/1858/

File: integration/spark-common/src/main/scala/org/apache/spark/util/FileUtils.scala [hunk 1]
```diff
@@ -73,7 +73,7 @@ object FileUtils {
       throw new DataLoadingException("Input file path cannot be empty.")
     } else {
       val stringBuild = new StringBuilder()
-      val filePaths = inputPath.split(",")
+      val filePaths = inputPath.split(",").map(_.trim)
```
Comment (Indhumathi27) [inline integration/spark-common/src/main/scala/org/apache/spark/util/FileUtils.scala]: Can you please add a testcase for this scenario

Comment (kevinjmh) [reply integration/spark-common/src/main/scala/org/apache/spark/util/FileUtils.scala]: done

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/224/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/1926/

**Comment by kevinjmh** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/226/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/1928/

**Comment by ajantha-bhat** [comment]
LGTM

**Pull request closed without merging**